docs/sources/tutorials/alerting-get-started-pt4/index.md

```diff
@@ -0,0 +1,407 @@
+  - [Git](https://git-scm.com/)
+
+### Set up the Grafana stack (OSS users)
+
+<!-- INTERACTIVE ignore END -->
+
+To demonstrate the observation of data using the Grafana stack, download and run the following files.
+
+1. Clone the [tutorial environment repository](https://www.github.com/grafana/tutorial-environment).
+
+   <!-- INTERACTIVE exec START -->
+
+   ```
+   git clone https://github.com/grafana/tutorial-environment.git
+   ```
+
+   <!-- INTERACTIVE exec END -->
+
+1. Change to the directory where you cloned the repository:
+
+   <!-- INTERACTIVE exec START -->
+
+   ```
+   cd tutorial-environment
+   ```
+
+   <!-- INTERACTIVE exec END -->
+
+1. Run the Grafana stack:
+
+   <!-- INTERACTIVE ignore START -->
+
+   ```
+   docker compose up -d
+   ```
+
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   <!-- INTERACTIVE exec START -->
+
+   ```bash
+   docker-compose up -d
+   ```
+
+   <!-- INTERACTIVE exec END -->
+
+   {{< /docs/ignore >}}
+
+   The first time you run `docker compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
+
+   <!-- INTERACTIVE ignore START -->
+
+   {{< admonition type="note" >}}
+   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
+   {{< /admonition >}}
+
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   NOTE:
+
+   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
+
+   {{< /docs/ignore >}}
+
+<!-- INTERACTIVE page step1.md END -->
+<!-- INTERACTIVE page step2.md START -->
+
+## How templating works
+
+In Grafana, you can use [templates](ref:templates) to dynamically pull in specific data about the alert rule. This results in more flexible and informative alert notification messages. You can template either alert rule labels and annotations, or notification templates. Both use the Go template language.
+
+{{< figure src="/media/docs/alerting/how-notification-templates-works.png" max-width="1200px" caption="How templating works" >}}
+
+### Templating alert rule labels and annotations
+
+[Labels and annotations](ref:template-labels-annotations) are key fields where templates are applied. Annotations are especially useful for providing additional context about the alert, such as a summary or description, which helps users quickly understand the alert rule’s meaning and cause. By using templating in annotations, you can customize the content of each alert instance, such as including instance names and metric values, so the notification becomes more informative.
```

Comment:
The main benefit of using templating in annotations is the ability to include query (dynamic) data.